packages/react-radio/src/components/RadioGroup/useRadioGroupStyles.ts

```diff
@@ -3,26 +3,27 @@ import { RadioGroupState } from './RadioGroup.types';
 
 export const radioGroupClassName = 'fui-RadioGroup';
 
-/**
- * Styles for the root slot
- */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
   },
 
-  // TODO add additional classes for different states and/or slots
+  horizontal: {
+    flexDirection: 'row',
+  },
 });
 
 /**
  * Apply styling to the RadioGroup slots based on the state
  */
-export const useRadioGroupStyles_unstable = (state: RadioGroupState): RadioGroupState => {
+export const useRadioGroupStyles_unstable = (state: RadioGroupState) => {
   const styles = useStyles();
-  state.root.className = mergeClasses(radioGroupClassName, styles.root, state.root.className);
-
-  // TODO Add class names to slots, for example:
-  // state.mySlot.className = mergeClasses(styles.mySlot, state.mySlot.className);
-
-  return state;
+  state.root.className = mergeClasses(
+    radioGroupClassName,
+    styles.root,
+    state.layout !== 'vertical' && styles.horizontal,
```

Comment:
`layout` can also be set to `horizontalStacked` but I don't see any handling of that option. Does that need to be implemented still? Should the `horizontalStacked` option be removed from the types?

Reply:
This line is handling both `horizontal` and `horizontalStacked` (`!== vertical`).

`horizontalStacked` is additionally handled in useRadio, which sets `labelPosition= 'below'`